operator/test/user.go

```diff
@@ -63,6 +63,7 @@ func CreateUser(
 ) *User {
 	ethClient, err := eth.CreateEthereumClient(ethURL, wallet, account)
 	if err != nil {
+		panic(err)
 		t.Fatal("creating ethereum wallet and client:", err)
```

Comment:
Now the `Fatal` call is not reached. `Fatal` would crash the application anyways, why the panic?